dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESIndexAPI.java

```diff
@@ -134,38 +134,46 @@ public String getStatus() {
 	}
 
 	public ESIndexAPI(){
-		this.esclient = new ESClient();
 		this.iapi = new ContentletIndexAPIImpl();
 		this.esIndexHelper = ESIndexHelper.INSTANCE;
 		this.serverAPI = APILocator.getServerAPI();
 		this.clusterAPI = APILocator.getClusterAPI();
 	}
 
 	@VisibleForTesting
-	protected ESIndexAPI(final ESClient esclient, final ContentletIndexAPIImpl iapi, final ESIndexHelper esIndexHelper,
+	protected ESIndexAPI(final ContentletIndexAPIImpl iapi, final ESIndexHelper esIndexHelper,
 						 final ServerAPI serverAPI, final ClusterAPI clusterAPI){
-		this.esclient = esclient;
 		this.iapi = iapi;
 		this.esIndexHelper = esIndexHelper;
 		this.serverAPI = serverAPI;
 		this.clusterAPI = clusterAPI;
 	}
 
-    /**
-     * returns all indicies and status
-     * @return
-     */
-    public Map<String,IndexStats> getIndicesAndStatus() {
-        final Client client = esclient.getClient();
-        final IndicesStatsResponse
-            indicesStatsResponse =
-                client.admin().indices().prepareStats().setStore(true).execute().actionGet(INDEX_OPERATIONS_TIMEOUT_IN_MS);
+	@SuppressWarnings("unchecked")
+	public Map<String, IndexStats> getIndicesStats() {
+		final Request request = new Request("GET", "/_stats");
+		final Map<String, Object> jsonMap = performLowLevelRequest(request);
 
-        return indicesStatsResponse.getIndices();
-    }
+		Map<String, IndexStats> indexStatsMap = new HashMap<>();
+
+		final Map<String, Object> indices = (Map<String, Object>)jsonMap.get("indices");
+
+		indices.forEach((key, value)-> {
+			Map<String, Object> indexStats = (Map<String, Object>) ((Map<String, Object>)
+					indices.get(key)).get("primaries");
+
+			int numOfDocs = (int) ((Map<String, Object>) indexStats.get("docs"))
+					.get("count");
+
+			int sizeInBytes = (int) ((Map<String, Object>) indexStats.get("store"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'sizeInBytes' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)